components/raftstore/src/store/entry_storage.rs

```diff
@@ -1177,7 +1177,20 @@ impl<EK: KvEngine, ER: RaftEngine> EntryStorage<EK, ER> {
                         } else {
                             range.1 == self.last_index() + 1
                         };
-                        assert!(is_valid, "the warmup range should still be valid");
+                        // FIXME: the assertion below doesn't hold.
+                        // assert!(is_valid, "the warmup range should still be valid");
+                        if !is_valid {
```

Comment:
Question: what's the impact when is_valid is false

Reply:
I think it's just an assumption about how async fetch log is used. When testing v1 this assertion failed. It's very rare so dropping the log entries should be fine.